app/Http/Controllers/Api/Device/DeviceHealthController.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace App\Http\Controllers\Api\Device;
+
+use App\Models\Device;
+use Illuminate\Http\Request;
+use App\Http\Controllers\Api\ApiController;
+
+class DeviceHealthController extends ApiController
+{
+    /**
+     * @api {get} /devices/:id/health Get Health Sensors
+     * @apiName Get_Device_Health
+     * @apiGroup Device Get all health sensors for a device
+     *
+     * @apiParam {Number} id Id of the Device
+     *
+     * @apiSuccessExample Success-Response:
+     *     HTTP/1.1 200 OK
+     *     {
+     *          "data":
+     *          [
+     *              {
+     *                  "sensor_class": "state",
+     *              },
+     *          ]
+     *     }
+     *
+     */
+    public function index(Device $device)
+    {
+        return $this->objectResponse($device->sensors()->deviceSensors());
+    }
+
+    /**
+     * @api {get} /devices/:id/health/:class Get all sensors for a Health Class
```

Comment:
Ignore my last comment as this would clash, however I think we do need an alternative route for getting graphs available to getting graphs.

Reply:
I was thinking all graph routes would be routed through a base route of /api/v1/graphs/....
Thoughts?